.gitignore

```diff
@@ -0,0 +1,17 @@
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Lock files
```

Comment:
Lock files should be comitted.

Reply:
When you use more than one kind of lock file, it may lead to conflict. The chosen one to be used for this repository was package-lock.json (npm), and the ignored one was yarn.lock (yarn)